TLSharp.Core/TelegramClient.cs

```diff
@@ -377,8 +377,15 @@ private void OnUserAuthenticated(TLUser TLUser)
             _session.SessionExpires = int.MaxValue;
 
             _session.Save();
-        }
-
+        }
+
+        public bool IsConnected()

```

Comment:
What do you mean? The method as it is now is not resource demanding so please make it a property. If in a second PR it becomes resource demanding, then convert it to a method in the next PR ;)